Doc/library/winreg.rst

```diff
@@ -488,7 +488,8 @@ This module offers the following functions:
    *type* is an integer that specifies the type of the data. See
    :ref:`Value Types <value-types>` for the available types.
 
-   *value* is a string that specifies the new value.
+   *value* is the new value to set. Accepts str, int, list of str, bytes-like object,
```

Comment:
I do not think it is useful to enumerate possible type without connection to the *type* value. Just say that acceptable types depend on the *type* value.